docs/data/routes/docs/nextjs/introduction/why-nextjs/en.md

```diff
@@ -3,4 +3,33 @@ name: why-nextjs
 routeTemplate: ./data/component-templates/article.yml
 title: Why Next.js and JSS
 ---
-# Why Next.js and JSS
\ No newline at end of file
+# Why Next.js and JSS?
+
+[Next.js](https://nextjs.org/) is a framework built on top of **React**. It aims to help developers create production-ready applications with minimal need for configuration and boilerplate code. 
+
+The latest JSS SDK leverages many of Next.js' [features](https://nextjs.org/#features), to simplify and enhance development workflow and experience. 
+
+## Improvements over previous JSS SDKs
+
+* Development and production runtime parity, including SSR during development.
+* Fewer confusing “modes” and deployment requirements.
+* No Headless SSR Proxy needed for production or for integration with Sitecore tracking.
+* No need to deploy your application bundle to the Content Management instance for Experience Editor support.
+
+## Features
+
+- Support for Next.js static HTML export, incremental static site generation, server-side rendering, and hybrid rendering.
```

Comment:
I know we link to the Next.js docs above, but I think it might be too much to expect that people will stop to go through all of those docs first. With that said, maybe we could either link these specific Next.js terms to specific pages in Next.js docs. Or (if there is time) what would be even easier to absorb is a simple, plain language sentence of what these are. I wish there was a way to do a helptext popover thing for these types of definitions.

Reply:
Oh, a popover would be lovely so we could add a quick explanation about such things. In the docs team, we look unkindly to having many links to external resources (we need to monitor other people's stuff to keep our docs current). 

Is your "Needs emojis :D" comment about the dryness of it all? If yes, thank you. If we pass documents through Grammarly, we aim for "neutral" tone (the straight face emoji)! 😆